build.gradle

```diff
@@ -38,6 +38,8 @@ dependencies {
 	compile "io.gitlab.arturbosch.detekt:detekt-rules:$detektVersion"
 	provided "org.sonarsource.sonarqube:sonar-plugin-api:$sonarVersion"
 
+	testCompile "junit:junit:4.12"
+	testCompile "com.nhaarman:mockito-kotlin-kt1.1:1.5.0"
 	testCompile "org.junit.jupiter:junit-jupiter-api:$junitEngineVersion"
```

Comment:
There is already junit5 und spek for testing, which I haven't used so far :/. I think you just have to change the imports for the annotations and they should work. I don't know if you are familiar spek testing. In case you want to try it out, but junit is ok too.

Reply:
I will update.